cmake/YarpOptions.cmake

```diff
@@ -239,6 +239,15 @@ endif()
 yarp_deprecated_option(YARP_FILTER_API) # Since YARP 2.3.68.1
 yarp_deprecated_option(YARP_CLEAN_API) # Since YARP 2.3.68.1
 
+
+#########################################################################
+# Enable/disable wanted, experimental, and unwanted warnings
+
+option(YARP_EXPERIMENTAL_API "enable unstable/experimental features")
+if(YARP_EXPERIMENTAL_API)
+  add_definitions(-DYARP_EXPERIMENTAL)
```

Comment:
I'd rather keep the option name and the definition synchronized, therefore I'd change this to 
```diff
- add_definitions(-DYARP_EXPERIMENTAL)
+ add_definitions(-DYARP_EXPERIMENTAL_API)
```